iroh/src/net_report.rs

```diff
@@ -700,30 +700,28 @@ async fn run_probe_v4(
     };
 
     let observer = Watchable::new(None);
-    let ob = observer.clone();
```

Comment:
what about probev6?

Reply:
Right! Sorry will do the same for that fn.